modules/acquia_cms_article/acquia_cms_article.install

```diff
@@ -40,3 +40,13 @@ function acquia_cms_article_update_8001() {
     $config->save();
   }
 }
+
+/**
+ * Implements hook_update_N().
+ *
+ * Make Display Author field optional.
+ */
+function acquia_cms_article_update_8002() {
+  $config = \Drupal::service('config.factory')->getEditable('field.field.node.article.field_display_author');
+  $config->set('required', '0')->save();
```

Comment:
@rajeshreeputra The value should be boolean . Right ?
Also, please use `FieldConfig` class to update config. So, it would be something like:
```suggestion
$field = FieldConfig::loadByName('node', 'article', 'field_display_author');
if ($field instanceof FieldConfig && $field->isRequired()) {
  $field->setRequired(FALSE);
  $field->save();
}
```

Reply:
updated